hassio/src/resources/hassio-style.js

```diff
@@ -5,7 +5,7 @@ documentContainer.setAttribute("style", "display: none;");
 
 export const hassioStyle = css`
   .card-group {
-    margin-top: 24px;
+    margin: 12px 0;
```

Comment:
So this impacts everywhere we use `card-group`. Have you audited this ?

Reply:
Ended up reverting this.